recipes/lorikeet/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+cp lorikeet-$PKG_VERSION.jar ${PREFIX}/bin
+
+cat <<END >${PREFIX}/bin/lorikeet
+#!/bin/sh
+
+JAR_PATH=`dirname \`which conda\``
+java ${JAVA_OPTS:-} -jar \$JAR_PATH/lorikeet-${PKG_VERSION}.jar \$*
```

Comment:
please follow the guidelines for Java packages: https://bioconda.github.io/guidelines.html#java